### PR TITLE
Task 4/show the chosen city weather

### PR DESCRIPTION
**Pull request by petyaNikleva**
Screenshot_3
Created new City page which shows the cities' current weather; Created new route for it.
Screenshot_1
Screenshot_2

### CODE REVIEW

File: src/components/constants.js [hunk 1]
```diff
@@ -2,7 +2,9 @@ export const baseUrl = 'https://pfa.foreca.com';
 export const serverURL = 'http://localhost:3030/auth';
 
 export const API_END_POINTS = {
```
Comment (virzhinia-petkova-redshelf) [inline src/components/constants.js:4]: good job with the object

File: src/pages/City/City.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+    getCityDetails(id)
+      .then(data => {
+        setCityName(data.name);
+        setCounrty(data.country);
+      })
+      .catch(() => setError(POSSIBLE_ERRORS.GET_CITY));
+  }, [id]);
+
+  let img = {};
```
Comment (virzhinia-petkova-redshelf) [inline src/pages/City/City.js]: I'm wondering why you're initializing your variable as an object if you're assigning it a string

**Review by virzhinia-petkova-redshelf — COMMENTED** [review]
looks good :)

File: src/components/constants.js (hunk 1 above)
Comment (petyaNikleva) [reply src/components/constants.js:4]: Thank you!

File: src/pages/City/City.js (hunk 2 above)
Comment (petyaNikleva) [reply src/pages/City/City.js]: My mistake. I changed the variable as a string. @virzhinia-petkova-redshelf

**Review by virzhinia-petkova-redshelf — APPROVED** [review]

**Pull request merged**